Makefile

```diff
@@ -1,4 +1,5 @@
 TEST_OPTIONS=-m unittest discover --start-directory tests --top-level-directory .
+CPU_COUNT=$(shell python -c "from multiprocessing import cpu_count; print(cpu_count())")
```

Comment:
I could write the following:

    CPU_COUNT=$(shell nproc)

But that approach raises significant portability issues.